src/ngraph/serializer.cpp

```diff
@@ -1450,6 +1451,13 @@ static shared_ptr<ngraph::Function>
                 node = make_shared<op::SpaceToDepth>(args[0], block_size);
                 break;
             }
+            case OP_TYPEID::Split:
+            {
+                const auto axis = node_js.at("axis").get<size_t>();
+                const auto splits = node_js.at("splits").get<vector<size_t>>();
```

Comment:
What about `num_split`?

Reply:
This param is only used in one of the constructors to populate the `m_splits` vector. Splitting the tensor is always done using values in that vector so there's no need to serialize nor deserialize this value.